llama_stack/providers/inline/inference/vllm/vllm.py

```diff
@@ -83,14 +83,14 @@ async def shutdown(self):
         if self.engine:
             self.engine.shutdown_background_loop()
 
-    async def register_model(self, model: ModelDef) -> None:
+    async def register_model(self, model: Model) -> None:
         raise ValueError(
             "You cannot dynamically add a model to a running vllm instance"
         )
 
-    async def list_models(self) -> List[ModelDef]:
+    async def list_models(self) -> List[Model]:
         return [
-            ModelDef(
+            Model(
                 identifier=self.config.model,
                 llama_model=self.config.model,
```

Comment:
return provider_id and provider_model_id as well

Reply:
I forgot to remove this. we no longer need to list the provider.